.github/workflows/first_interaction.yml

```
@@ -0,0 +1,16 @@
+name: First interaction
+
+on:
+  pull_request:
+  issues:
+
+jobs:
+    name: Run
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: TylerLeonhardt/first-interaction-pwsh@master
```

```suggestion
    - uses: TylerLeonhardt/first-interaction-pwsh@v0.1.0
```

There is no master tag: https://github.com/TylerLeonhardt/first-interaction-pwsh/tags

---

Hm looks like one could use master as well :thinking: 
But if there are breaking changes, it will break our greeter. Always use fixed versions :wink: